src/utils/validators.js

```diff
@@ -76,12 +76,12 @@ const HERO_DROPDOWN_MAX_LENGTH = 30;
 // Contact Us Editor
 // ===============
 // Contacts
-const CONTACT_TITLE_MIN_LENGTH = 2;
+const CONTACT_TITLE_MIN_LENGTH = 0;
 const CONTACT_TITLE_MAX_LENGTH = 30;
 const CONTACT_DESCRIPTION_MAX_LENGTH = 400;
 
 // Locations
-const LOCATION_TITLE_MIN_LENGTH = 2;
+const LOCATION_TITLE_MIN_LENGTH = 0;
```

Comment:
Nit: I feel like these should be min_length 1 and the checks left as `value.length < CONTACT_TITLE_MIN_LENGTH` so that the wording is clearer

Reply:
good idea eb4b82c